src/components/paneler/innhold/personaliaV2/adresser.tsx

```diff
@@ -3,114 +3,214 @@ import { isNullOrUndefined, visEmdashHvisNull } from '../../../../utils';
 import { Element, Normaltekst } from 'nav-frontend-typografi';
 import EMDASH from '../../../../utils/emdash';
 import {
+	Kontaktadresse,
 	Matrikkeladresse,
-	PersonaliaMidlertidigAdresseUtland,
-	PersonaliaPostadresse,
 	PersonaliaV2Info,
+	PostadresseIFrittFormat,
+	Postboksadresse,
+	Ukjentbosted,
+	Utenlandskadresse,
+	UtenlandskadresseIFrittFormat,
 	Vegadresse
 } from '../../../../rest/datatyper/personaliav2';
-import { OrNothing } from '../../../../utils/felles-typer';
+import {OrNothing} from '../../../../utils/felles-typer';
 
-function SammensattFolkeregistrertAdresse(props: Pick<PersonaliaV2Info, 'bostedsadresse'>) {
+function BostedsAdresse(props: Pick<PersonaliaV2Info, 'bostedsadresse'>) {
 	if (isNullOrUndefined(props.bostedsadresse)) {
 		return null;
 	}
 
 	const vegadresse = props.bostedsadresse && props.bostedsadresse.vegadresse as Vegadresse;
 	const matrikkeladrese = props.bostedsadresse && props.bostedsadresse.matrikkeladresse as Matrikkeladresse;
+	const utenlandskadresse = props.bostedsadresse && props.bostedsadresse.utenlandskAdresse as Utenlandskadresse;
```

Comment:
I typescript 4 så er det mulig å bruke ?. syntax for å hente ut verdier fra objekter på en enklere måte.
F.eks `props.bostedsadresse?.vegadresse as Vegadresse`

Reply:
Strengt talt ikke så viktig, men koden blir lettere å lese :)

Reply:
Siden det blir gjort null-sjekker på alle variablene, så trenger man ikke `as Vegadresse` osv. F.eks 
`const vegadresse =  props.bostedsadresse?.vegadresse;` holder :)